comixed-web/src/assets/i18n/en/library.json

```diff
@@ -102,6 +106,16 @@
       "download-comic": "Download this comic...",
       "go-to-next-comic": "Go to the next comic in this series...",
       "go-to-previous-comic": "Go to the previous comic in this series..."
+    },
+    "comic-effects": {
```

Comment:
In the web project, I've been doing things differently than in the frontend project (learned some good lessons). For example, everything to do with scraping a comic is under the "scraping" group in the translations.

Since this is related to a single comic, can we move these translations to a group named "comic"? So it would be (without the comments, of course :smile:):

```
{
  "comic": { // the grouping 
    "save-comic": {  // the specific feature in that group
      "effect-success": "Changes successfully saved.",
      "effect-failure": "Failed to save changes."
    }
  }
```

Using "effect-success" and "effect-failure" keeps things simple when searching for an entry.